docs/guides/src/main/server/db.adoc

```diff
@@ -90,7 +90,9 @@ For details on these properties and any performance implications, see the Oracle
 
 === Unicode support for a Microsoft SQL Server database
 
-Unicode characters are supported only for the special fields for a Microsoft SQL Server database. The JDBC driver and database require no special settings.
+Unicode characters are supported only for the special fields for a Microsoft SQL Server database. The database requires no special settings,
+but the `sendStringParametersAsUnicode` property needs to be set to `false` in the JDBC driver. This is needed to prevent deadlocks that can
+happen during data type conversion.
```

Comment:
```suggestion
Unicode characters are supported only for the special fields for a Microsoft SQL Server database. The database requires no special settings.

The `sendStringParametersAsUnicode` property of JDBC driver should be set to `false` to significantly
improve performance. Without this parameter, the Microsoft SQL Server might be unable to use indexes.
```

Reply:
That does look a lot better. Thanks @hmlnarik , updated the PR as per your suggestion.